data/pc/common/features.json

```diff
@@ -339,6 +339,156 @@
     "description": "sign text send when updating signs is send as stringified strings",
     "versions": ["1.8", "1.8.9"]
   },
+  {
+    "name": "usesBlockStates",
+    "versions": ["1.13", "1.19"]
+  },
+  {
+    "name": "effectNamesMatchRegistryName",
+    "versions": ["1.17", "1.19"]
+  },
+  {
+    "name": "shieldSlot",
+    "versions": ["1.9", "1.19"]
+  },
+  {
+    "name": "village&pillageInventoryWindows",
+    "versions": ["1.14", "1.19"]
+  },
+  {
+    "name": "netherUpdateInventoryWindows",
+    "versions": ["1.16","1.19"]
+  },
+  {
+    "name": "unloadChunkByEmptyChunk",
+    "description": "Chunk unloading is done by sending an empty chunk",
+    "versions": ["1.8", "1.8"]
+  },
+  {
+    "name": "unloadChunkDirect",
+    "description": "Chunk unloading is done by sending directly an unload chunk packet",
+    "versions": ["1.9", "1.16"]
+  },
+  {
+    "name": "fixedPointPosition",
```

Comment:
I think some of those were already present in the file
Can you check and add only the new ones ?

Reply:
I double checked it and couldn't find any duplicates sorry

Reply:
do you mean the ones I copy pasted from flying squid features.json?

Reply:
Oh I see it now. Working on it

Reply:
Done. No more duplicates